docs/help-topics/rhtap-secrets/rhtap-secrets.yml

```diff
@@ -0,0 +1,44 @@
+# Name is an internal name. Title displays in the UI as a side panel title.
+# Leave tags empty for now. Tags will specify where in the app descriptions will be available.
+# Links to be external only. We don't know yet whether referencing other side panels will be supported but referencing to in-depth docs is expected to be supported.
+# Note that this topic is for LEARN MORE.
+
+- name: rhtap-secrets-secrets
+  tags:
+  title: Secrets
+  content: |-
+    Add and manage your build and deployment secrets and their related configurations. You can add a secret at the workspace level to keep your data private. All secrets are stored using AWS Secrets Manager.  
+
+    ## Secrets storage in AWS Secrets Manager
+    All of your secrets are stored in AWS Secrets Manager to make sure that only applications within your workspace can access them. For privacy, avoid using environment variables to store your secrets because they send the contents of the secret back to publicly accessible repositories. 
```

Comment:
Suggestion: "... to make sure..." --> "so that" 
reason: shorter and less complex 

Question: are we supposed to use "within" with workspace? "In" would be less complex 

"For privacy" --> What about privacy? Maybe: "To keep your application secure..." or something similar 

Suggested edit: "Avoid environment variables because they store your secrets and then send the secret's contents back to..."